.gitignore

```diff
@@ -0,0 +1 @@
+**/.idea/
```

Comment:
Hey there, do you mind deleting this file from the PR?

I highly recommend setting up a [global `.gitignore`](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/) to ignore these files from all git projects without needing to set it up in each project!

Reply:
Done.
Just have to note that, in my opinion, having `.gitignore` within your repo ensures consistency of commits without relying on environment external to the project.